src/Tribe/Tickets.php

```diff
@@ -2842,6 +2842,9 @@ public function ticket_add( $post_id, $data ) {
 				$ticket->end_date = date( Tribe__Date_Utils::DBDATEFORMAT, strtotime( $end_datetime ) );
 			}
 
+			// Pass the control to the child object
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 1e0a8762b1c11a6aae6293cca2eb0fe9</sub>